### PR TITLE
Skill and evaluation controllers implemention

### PR DESCRIPTION
**Pull request by Evgeniy-Sinicin**
... also added dtos to feedback controller

### CODE REVIEW

File: InternshipBE/BL/Interfaces/IEvaluationService.cs [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace BL.Interfaces
+{
+    public interface IEvaluationService
+    {
+        Task<CreateEvaluationDTO> CreateEvaluationAsync(CreateEvaluationDTO createEvaluationDto);
+
+        Task<List<StackGetEvaluationDTO>> GetEvaluationsByFeedbackId(int feedbackId);
```
Comment (romanstat) [inline InternshipBE/BL/Interfaces/IEvaluationService.cs]: Async

File: InternshipBE/DAL/Repositories/FeedbackRepository.cs [hunk 2]
```diff
@@ -14,5 +13,32 @@ public class FeedbackRepository : GenericRepository<Feedback>, IFeedbackReposito
...
+                .ThenInclude(x => x.Skill)
+                .Where(x => x.CandidateId == id)
+                .ToListAsync();
+        }
+
+        public async override Task<Feedback> UpdateAsync(Feedback feedback)
+        {
+            await Task.Run(() => _context.Evaluations.UpdateRange(feedback.Evaluations));
+            
+            return await base.UpdateAsync(feedback);
```
Comment (romanstat) [inline InternshipBE/DAL/Repositories/FeedbackRepository.cs]: looks not good, is it okey?

File: InternshipBE/BL/DTOs/CandidateDTOs/CandidateDTO.cs [hunk 3]
```diff
@@ -21,9 +20,9 @@ public class CandidateDTO
 
         public string Skype { get; set; }
 
-        public StackType StackType { get; set; }
+        public string StackType { get; set; }
 
-        public EnglishLevelType EnglishLevelType { get; set; }
+        public string EnglishLevelType { get; set; }
```
Comment (Amarozka123) [inline InternshipBE/BL/DTOs/CandidateDTOs/CandidateDTO.cs]: Rename all EnglishLevelType properties to EnglishLevelName

File: InternshipBE/BL/DTOs/CountryDTOs/CountryDTO.cs [hunk 4]
```diff
@@ -1,4 +1,4 @@
-﻿namespace BL.DTOs
+﻿namespace BL.DTOs.CountryDTOs
```
Comment (romanstat) [inline InternshipBE/BL/DTOs/CountryDTOs/CountryDTO.cs:1]: if you separate to .CountryDTOs, so for all controllers services create folders

Comment (romanstat) [reply InternshipBE/BL/DTOs/CountryDTOs/CountryDTO.cs:1]: if there are such places

**Review by romanstat — APPROVED** [review]

**Review by Xodoc — APPROVED** [review]

**Review by vldmrklnk — APPROVED** [review]

**Pull request merged**